src/scss/components/_buttons.scss

```
@@ -180,5 +180,23 @@ $button-colors: 'error' !default;
       margin-right: var(--spacer-x-small);
     }
   }
+
+  &.chevron-down::after {
+    content: '';
+    width: 20px;
+    height: 14px;
+    background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAiIGhlaWdodD0iMTIiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTE4LjIxNCAwIDEwIDguMzY0IDEuNzg2IDAgMCAxLjgxOCAxMCAxMiAyMCAxLjgxOCAxOC4yMTQgMFoiIGZpbGw9IiMzRjNFM0UiLz48L3N2Zz4=");
+    background-repeat: no-repeat;
+  }
```

You can use the svg directly in content, instead of using a more hackish solution with backgrounds.

```suggestion
    content: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAiIGhlaWdodD0iMTIiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTE4LjIxNCAwIDEwIDguMzY0IDEuNzg2IDAgMCAxLjgxOCAxMCAxMiAyMCAxLjgxOCAxOC4yMTQgMFoiIGZpbGw9IiMzRjNFM0UiLz48L3N2Zz4=");
  }
```